Automattic/wp-calypso
# Jetpack Cloud: Use Site Capabilities API to determine service level

#### Changes proposed in this Pull Request

* This PR adds redux state for getting and storing site capabilities in `state.rewind[siteId].capabilities`. This is stored as a list that can be queried with `_.includes` for specific capabilities, including realtime backup.
* We are also adapting the Backup section main component to consume this list, and provide service level to the child components using this new API, rather than querying site purchases, which isn't necessarily accurate for whether a site can do realtime backups or not.

#### Testing instructions

* Load up Jetpack Cloud Dev Backups section for a site which has realtime backups. Query `state.rewind[siteId].capabilities` and ensure that a list is loaded into state of the site's capabilities.
* Do the same for a site with daily backups. The list should now represent daily backups.
* The UI should change (slightly) depending on whether you are viewing a realtime or daily backup site.
* Obviously, be diligent in code review since this touches the main calypso reducer and data-layer.

Fixes #
